db/migrate/20170409170753_add_last_webfingered_at_to_accounts.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastWebfingeredAtToAccounts < ActiveRecord::Migration[5.0]
+  def change
+    add_column :accounts, :last_webfingered_at, :datetime, null: true, default: nil
```

Comment:
If I understand https://www.postgresql.org/docs/9.1/static/datatype-datetime.html correctly, in Postgresql, those two should be the same thing. Furthermore, it seems only datetime is currently in Mastodon, not timestamp.